README.md

```diff
@@ -82,9 +82,9 @@ await keyPair.export({publicKey: true});
 // ->
 {
   type: 'Multikey',
-  id: 'did:example:1234#z6Mkon3Necd6NkkyfoGoHxid2znGc59LU3K7mubaRcFbLfLX',
+  id: 'did:example:1234#zynkWQ2LUf58H4VwLGjscwb9KJwGUmqBFXdJKnCXxDMciFXioY7Hq7MvEdVfsBiQSVa9k9',
```

Comment:
Ensure that the fingerprint corresponds to a generated keypair. The header for the curves should be the following (as of [962df2013b](https://github.com/digitalbazaar/ecdsa-multikey/tree/962df2013baec09d39fa66fa894436edf1b027ec)):


```
"zDnae..." => P-256
"z82Lk..." => P-384
"z2J9g..." => P-521
```